config/job_conf.xml.sample_advanced

```diff
@@ -204,6 +217,37 @@
                  This option sets a limit of retrials, so that after that number of failed pods, the job is re-scaled to
                  zero (no execution) and the stderr/stdout of the k8s job is reported in galaxy (and the galaxy job set
                  to failed) -->
+
+            <param id="k8s_supplemental_group_id">0</param>
+            <param id="k8s_fs_group_id">0</param>
+            <!-- If mounting an NFS / GlusterFS or other shared file system which is administered to ONLY provide access
+                 to a DEFINED user/group, these variables set the group id that Pods need to use to be able to read and
+                 write from that mount. If left to zero or deleted, these parameters are neglected. Integer values
+                 above zero trigger the addition of a security context on each Pod created to dispatch jobs:
+
+                 securityContext:
+                    supplementalGroups: [value-set-goes-here]
+                    fsGroup: fs-group-integer-value
+
+                 inside the Pod spec, applicable for all containers on the pod. Just one of them set to >0 will generate
+                 the security context.
+
+                 Using this requires that the Kubernetes cluster is not running the admission controller
+                 "SecurityContextDeny". To check this, look at the --admission-control= variable setup for the
```

Comment:
`--` is not allowed inside XML comments, see https://stackoverflow.com/questions/10842131/xml-comments-and